digits/dataset/images/classification/forms.py

```diff
@@ -21,13 +21,16 @@ class ImageClassificationDatasetForm(ImageDatasetForm):
                                   choices=[
                                       ('lmdb', 'LMDB'),
                                       ('hdf5', 'HDF5'),
+                                      ('tfrecords', 'TFRecords'),
```

Comment:
You said previously to keep it in case some others want to extend it and get it working?